docs/use_cases/recommender_systems.md

```diff
@@ -1,32 +1,31 @@
 
-However, it is sometimes the case that only scarce side information about the users or the items is available to us. Consider, for example, a public service provider that has its own audio- and video-on-demand platform; such companies have restrictions with regard to collecting/storing user metadata. But even in this case, we still have useful information about the items that are served, such as genre, popularity, etc. Still, developers often disregard this side information, because it is scarce, and proceed with collaborative-filtering (CF) approaches, which make use of the historical interactions only (data consisting of user-item pairs). While CF work reasonably well in this use case - extrapolate user preferences via similarities of all-user’s browsing/purchasing history - one question remains: can we improve on the quality by (somewhow) making use of available information, regardless how scarce? More precisely, can we use the collaborative filtering paradigm in this case, with this scarce side info? Yes, there are libraries that allow us to ‘inject’ side information ([LightFM](https://making.lyst.com/lightfm/docs/home.html), for example). But there is also a way to extend the most efficient and effective collaborative filtering models, such as [ALS Matrix Factorization (MF)](http://yifanhu.net/PUB/cf.pdf) (the Python [‘implicit’](https://github.com/benfred/implicit) library) or EASE, to make use of the side information.
+However, it is sometimes the case that only scarce metadata about the users or the items is available to us. Public service providers, with their own on-demand audio and video platform, for example, are restricted in collecting user metadata. Such companies still have item data - genre, popularity, and so on - that can be used to improve the quality of the recommendations. Still, developers often disregard this side information, because it is scarce, and proceed with collaborative-filtering (CF) approaches, which make use of the historical interactions only (data consisting of user-item pairs). While CF work reasonably well in this use case - extrapolate user preferences via similarities of all users' browsing/purchasing history - can we improve the quality of the recommendations (increase users' engegement) using available side information even when it's scarce? More precisely, can we use the collaborative filtering paradigm in this case, with this scarce side info? Yes, there are libraries that allow us to ‘inject’ side information ([LightFM](https://making.lyst.com/lightfm/docs/home.html), for example). But there is also a way to extend the most efficient and effective collaborative filtering models, such as [ALS Matrix Factorization (MF)](http://yifanhu.net/PUB/cf.pdf) (the Python [‘implicit’](https://github.com/benfred/implicit) library) or [EASE] (https://arxiv.org/abs/1905.03375) (Embarassingly Shallow Autoencoder), to make use of the side information.
```

Comment:
I changed this

instead of "and proceed with collaborative... ...pairs"
say "Typically, developers use collaborative-filtering (CF) approaches, which employ historical interactions (data consisting of user-item pairs) to extrapolate user preferences via simliiarities of all users' browsing /purchasing history."
*and move the latter sentence up to right after "...collecting user metadata."

But the part '*and move...' I don't understand

Reply:
I've made the changes, but this paragraph has a lot of redundant info, in my opinion

Reply:
move "Typically, developers use collaborative-filtering (CF) approaches, which employ historical interactions (data consisting of user-item pairs) to extrapolate user preferences via simliiarities of all users' browsing /purchasing history."
so that it comes right after "Public service providers, with their own on-demand audio and video platform, for example, are restricted in collecting user metadata."

Reply:
done